src/sql/azdata.d.ts

```diff
@@ -91,6 +92,7 @@ declare module 'azdata' {
 			groupId: string;
 			saveProfile: boolean;
 			azureTenantId?: string;
+			azureAccount?: string;
```

Comment:
all the changes to this file should go into `azdata.proposed.d.ts`.  We move into here after the changes have been shipped in a couple releases and proven to meet the desired functionality.

Reply:
How does this work with the other code in ADS? When it imports from azdata, is that imported from azdata.proposed.d.ts? If it is, then I don't have a problem moving it.

Reply:
Please let me know about how azdata.proposed.d.ts works in the code because there are some types in azdata that need to be modified. The modifications add optional members, but not entirely new types. So, I'm not clear on how contents of azdata.d.ts and azdata.proposed.d.ts mix. Please explain, or point me to some documentation that explains.

Reply:
code that imports from both azdata and azdata proposed get a 'azdata' module that is a union of the two.

Reply:
I just read up on combining declarations in Typescript, and will make the requested code changes as possible. Thanks.

Reply:
All changed to azdata.d.ts reverted and moved to azdata.proposed.d.ts.